packages/ckeditor5-dev-tests/lib/utils/fileoptiontoglob.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const nodeModulesPath = path.join( process.cwd(), 'packages' );
+
+/**
+ * Converts values of --files argument to proper globs.
+ * There are 5 supported types of values now:
+ *
+ * 0. current package's tests (when run in context of a package – e.g. on CI) - '/'
+ * 1. all packages' files – '*'
+ * 2. given package files – 'engine'
+ * 3. everything except the given package – '!engine'
+ * 4. path – 'engine/view' -> 'ckeditor5-engine/tests/view/**\/*.js'
+ * 5. simplified glob – 'engine/view/**\/*.js' -> 'ckeditor5-engine/tests/view/**\/*.js'
+ *
+ * @param {String} file
+ * @param {Boolean} isManualTest=false
+ * @returns {String}
+ */
+module.exports = function fileOptionToGlob( file, isManualTest = false ) {
```

Comment:
I'm not sure, but AFAIK the default values for parameters should be in the square brackets.